### PR TITLE
DOCS-6279 Adds Getting Started with CI Visibility Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Adds a Getting Started with CI Visibility doc page + screenshots to the Getting Started section of the Docs Site.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/getting_started/ci_visibility/_index.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+CI Visibility provides DevOps and platform engineering organizations with comprehensive monitoring, analytics, and the ability to pinpoint and resolve bottlenecks, optimize resource allocation, and decrease pipeline costs. 
+
+By integrating performance metrics, logs, and alerts, your organization can maintain development speed, improve the reliability of your pipelines, and make data-informed decisions across cloud and self-hosted environments.
+
+## Set up your CI provider
+
+CI Visibility tracks the performance and results of your CI pipelines, and displays results after the pipeline finishes. 
+
+To start sending pipeline metrics, see the documentation for one of the following CI providers.
```
[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/getting_started/ci_visibility/_index.md:39]: Should we mention what they can do if Datadog does not support their CI provider? Manual steps, if any?

File: content/en/getting_started/ci_visibility/_index.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+[6]: https://app.datadoghq.com/dash/integration/30516/ci-visibility---pipelines-dashboard
+[7]: https://app.datadoghq.com/ci/pipelines
+[8]: /continuous_integration/search/
+[9]: /continuous_integration/explorer?tab=pipelineexecutions
+[10]: /tests/guides/flaky_test_management/
+[11]: /continuous_integration/pipelines/?tab=githubactions#supported-features
+[12]: /continuous_integration/guides/infrastructure_metrics_with_gitlab/
+[13]: /continuous_integration/guides/pipeline_data_model/
+[14]: /monitors/types/ci/?tab=pipelines
+[15]: https://app.datadoghq.com/monitors/create/
```
Comment (rtrieu) [inline content/en/getting_started/ci_visibility/_index.md:158]: Link doesn't work when there is a slash at the end

```suggestion
[15]: https://app.datadoghq.com/monitors/create
```

File: content/en/getting_started/ci_visibility/_index.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+{{< img src="/getting_started/ci_visibility/avg_duration_monitor.png" alt="A CI pipeline monitor configured to trigger an alert when the average duration for the Test and Deploy Cart pipeline exceeds five minutes in the past day" style="width:100%" >}}
+
+To set up a monitor that alerts on your CI pipeline when the average duration in the past day exceeds a five minute threshold:
+
+1. Navigate to [**Monitors** > **New Monitor**][15] and select **CI**. 
+1. Select a common monitor type for CI pipelines to get started, for example: `Long Running Pipeline` to trigger alerts when a pipeline has been running for too long or `Failed Job` to trigger alerts for job failures, or customize your own search query. In this example, enter `@ci.pipeline.name:test_and_deploy_cart` and select the Avg of `Duration (@duration)`.
+1. In the `Evaluate the query over the` section, select **last 1 day**. 
+1. Set the alert conditions to trigger when the evaluated value is **above** the threshold, and specify values for the alert or warning thresholds, such as `Alert threshold > 300000000000`.
+1. Define the monitor's notification.
```
Comment (rtrieu) [inline content/en/getting_started/ci_visibility/_index.md:136]:
```suggestion
1. In the `Notify your team` section, configure your monitor's notification settings.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Left some feedback for your review, but otherwise looks good!

File: content/en/getting_started/ci_visibility/_index.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+CI Visibility tracks the performance and results of your CI pipelines, and displays results after the pipeline finishes. 
+
+To start sending pipeline metrics, see the documentation for one of the following CI providers. 
+
+{{< partial name="getting_started/ci_visibility/providers.html" >}}
+
+</br>
+
+Depending on the CI provider(s) of choice, CI Visibility may not support all of the levels in your pipeline (stage, job, step, or command). For more information about how CI Visibility defines a CI pipeline, see the [Terminology section][4].
```
Comment (alai97) [inline content/en/getting_started/ci_visibility/_index.md:45]:
```suggestion
For a CI provider that is not supported by Datadog, you can send your pipeline events programmatically using the [CI Visibility Pipelines API].

Depending on the CI provider(s) of choice, CI Visibility may not support all of the levels in your pipeline (stage, job, step, or command). For more information about how CI Visibility defines a CI pipeline, see the [Terminology section][4].
```

File: content/en/getting_started/ci_visibility/_index.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+CI Visibility tracks the performance and results of your CI pipelines, and displays results after the pipeline finishes. 
+
+To start sending pipeline metrics, see the documentation for one of the following CI providers that Datadog supports below.
+
+{{< partial name="continuous_integration/ci-pipelines-getting-started.html" >}}
+
+</br>
+
+If your CI provider is not supported, you can programmatically send your pipeline events to Datadog using the [CI Visibility Pipelines API][16].
```
Comment (alai97) [inline content/en/getting_started/ci_visibility/_index.md:45]:
```suggestion
If your CI provider is not supported, you can programmatically send your pipeline events to Datadog. See [Send pipeline events to Datadog](#send-pipeline-events-to-datadog).
```

File: content/en/getting_started/ci_visibility/_index.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+After you have set up Test Visibility, you can access information about tests that were run in your CI pipelines, including the test status (Failed, New Flaky, Passed, or Skipped), on the Test Runs tab in a pipeline execution’s side panel. For more information, see the [Flaky Test Management documentation][10].
+
+You can access pipeline or job logs across cloud and self-hosted runners and see information about your runners on the Logs tab in a pipeline execution’s side panel.
+
+If you are using [supported providers][11], you can correlate infrastructure metrics with your GitLab jobs and access the GitLab job’s host, system, host tags, and host metrics information. For more information, see [Correlate Infrastructure Metrics with GitLab Jobs in Datadog][12].
+
+## Send pipeline events to Datadog
+
+For other pipeline providers and custom pipelines, you can programmatically send pipeline events to Datadog using the CI Visibility Pipelines API endpoint. For more information, see [Pipeline Data Model and Execution Types][13].
```
Comment (alai97) [inline content/en/getting_started/ci_visibility/_index.md:122]:
```suggestion
For other pipeline providers and custom pipelines, you can programmatically send pipeline events to Datadog using the [CI Visibility Pipelines API endpoint][16]. For more information, see [Pipeline Data Model and Execution Types][13].
```

**Review by rtrieu — APPROVED** [review]
Looks good!

**Review by KickingTheTV — APPROVED** [review]

**Pull request merged**